### ISSUE TITLE
Missing title for the View Team Assessmnents

### ISSUE THREAD

**Issue by LaurentGoderre** [wet-boew/web-reporting#31]
It should be the team name followed by "Assessment Scores". For example: Global Missions Publishing Assessment Scores

![missing_title](https://f.cloud.github.com/assets/1090826/17162/160645c6-486a-11e2-819f-f0f1599789d5.JPG)
